src/hotspot/share/utilities/resourceHash.hpp

```diff
@@ -77,16 +78,22 @@ class ResourceHashtable : public ResourceObj {
 
   Node const** lookup_node(unsigned hash, K const& key) const {
     return const_cast<Node const**>(
```

Comment:
[pre-existing] I think this `const_cast` to add const is unnecessary.

Reply:
I tried removing it but gcc complains. I will leave it as is for now.

Reply:
Oops, I misread it.  OK.